### DISCUSSION TITLE
Equivalent function of cv2.fillConvexPoly in Kornia

### DISCUSSION THREAD

**Discussion by vipulkumar-developer** [kornia/kornia#1599] [Questions]
Hi, I've started a new discussion to ask if there exists a method like cv2.fillConvexPoly in Kornia.

**Comment by edgarriba** [comment]
@vipulkumar-developer we don't have such functionality yet
I'll open a ticket pointing here /cc @shijianjian 

We will be more than happy to receive a contribution in that direction :)
